### PR TITLE
style(ui-ux): update design of prices

### LINKED ISSUE #2595: `Liquidity Pages` - Price Details
#### What would you like to be added:
For Add liquidity and remove liquidity price details row. Use same format as DEX 
Screenshot_20220420-143924_DeFiChain Wallet

### PR DESCRIPTION
**Pull request by pierregee**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- Update the design of prices section - add/remove liquidity (to be consistent with swap screen)
- Create function for repetitive tests related to prices
- Move the PriceSection component from DEX folder to components
- Move the translations of PriceSection on its own

#### Which issue(s) does this PR fixes?:

Fixes #2595

#### Additional comments?:
- Future swap feature has some updates in price rates section for swap. Thus, it would be inconsistent again once it's been released. We'll iterate over other pages once the Future Swap has been released

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ import { ConversionInfoText } from '@components/ConversionInfoText'
 import { FeeInfoRow } from '@components/FeeInfoRow'
 import { InputHelperText } from '@components/InputHelperText'
 import { NumberRow } from '@components/NumberRow'
-import { PriceRateProps, PricesSection } from './components/PricesSection'
+import { PriceRateProps, PricesSection } from '../../../../../components/PricesSection'
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx:33]: can use `@components/PricesSection`

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { ThemedIcon, ThemedScrollView, ThemedSectionTitle, ThemedView } from '@c
 import { TextRow } from '@components/TextRow'
 import { NumberRow } from '@components/NumberRow'
 import { FeeInfoRow } from '@components/FeeInfoRow'
-import { PricesSection } from './components/PricesSection'
+import { PricesSection } from '../../../../../components/PricesSection'
```
Comment (fullstackninja864) [inline mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx:20]: can use `@components/PricesSection`

File: shared/translations/languages/de.json [hunk 3]
```diff
@@ -1339,5 +1336,10 @@
...
   "components/Announcements": {
     "DETAILS": "DETAILS"
+  },
+  "components/PricesSection": {
+    "PRICES": "PREISE",
+    "{{tokenA}} price in {{tokenB}}": "{{tokenA}}-Preis in {{tokenB}}",
+    "{{tokenB}} price in {{tokenA}}": "{{tokenB}}-Preis in {{tokenA}}"
   }
-}
+}
```
Comment (fullstackninja864) [inline shared/translations/languages/de.json:1345]: please add new line

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx (hunk 2 above)
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx:20]: Updated!

File: mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx (hunk 1 above)
Comment (pierregee) [reply mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreen.tsx:33]: Updated!

File: shared/translations/languages/de.json (hunk 3 above)
Comment (pierregee) [reply shared/translations/languages/de.json]: Updated!

**Review by fullstackninja864 — DISMISSED** [review]

**Review by thedoublejay — CHANGES_REQUESTED** [review]
Prices are displayed wrong.

**Comment by pierregee** [comment]
> Prices are displayed wrong.

Updated!

**Pull request merged**